iso/scripts/generate_dappnode_iso_debian.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
 set -e
 
-# Source = https://cdimage.debian.org/cdimage/unofficial/non-free/cd-including-firmware/current/amd64/iso-cd/
-# updated permalink to archive of debian ISO: https://cdimage.debian.org/cdimage/unofficial/non-free/cd-including-firmware/archive/11.5.0+nonfree/amd64/iso-cd/
-ISO_NAME=firmware-11.5.0-amd64-netinst.iso
+# Source = https://cdimage.debian.org/cdimage/unofficial/non-free/cd-including-firmware/current/amd64/iso-cd/ 
+# https://cdimage.debian.org/cdimage/unofficial/non-free/cd-including-firmware/current/amd64/iso-cd/firmware-11.7.0-amd64-netinst.iso
+# updated permalink to archive of debian 11.5 ISO: https://cdimage.debian.org/cdimage/unofficial/non-free/cd-including-firmware/archive/11.5.0+nonfree/amd64/iso-cd/
+ISO_NAME=firmware-11.7.0-amd64-netinst.iso
 ISO_PATH="/images/${ISO_NAME}"
-ISO_URL=https://cdimage.debian.org/cdimage/unofficial/non-free/cd-including-firmware/archive/11.5.0+nonfree/amd64/iso-cd/
-SHASUM="ce1dcd1fa272976ddc387554202013e69ecf1b02b38fba4f8c35c8b12b8f521e  ${ISO_PATH}"
+ISO_URL=https://cdimage.debian.org/cdimage/unofficial/non-free/cd-including-firmware/current/amd64/iso-cd/
```

Comment:
```suggestion
ISO_URL=https://cdimage.debian.org/cdimage/unofficial/non-free/cd-including-firmware/11.7.0+nonfree/amd64/iso-cd/
```

We shouldn't use current because current changes. :)

Reply:
thank you i was writing this out in my last comment but then noticed your collapsed commit and edited my post since i asked if there was a permalink already.